### ISSUE TITLE
color.pathway.by.objects()' failed to extract KEGG image path from response

### ISSUE THREAD

**Issue by apelin20** [Bioconductor/KEGGREST#12]
Hello,

Can't seem to get color.pathway.by.objects() to work and it is required by pathfindR.

```
color.pathway.by.objects("path:hsa00010", c("hsa:2821", "hsa:226"), c("#ff0000", "#00ff00"), c("#ffff00", "yellow"))
Error in color.pathway.by.objects("path:hsa00010", c("hsa:2821", "hsa:226"),  : 
  'color.pathway.by.objects()' failed to extract KEGG image path from response.
```

And yet this works

```
png <- keggGet("hsa05130", "image") 
writePNG(png, "hsa05130_tmp.png")
```

I am not sure why this is happening. In pathfindR I am having the following error:

```
> visualize_hsa_KEGG("hsa00010", input_processed = input2)
Downloading pathway diagrams of 1 KEGG pathways

  |                                                                      |   0%Cannot retrieve PNG url: hsa00010
Here's the original error message:
'color.pathway.by.objects()' failed to extract KEGG image path from response.
Cannot download PNG file: NA
Here's the original error message:
'url' must be a length-one character vector
  |======================================================================| 100%
Saving colored pathway diagrams of 0 KEGG pathways

Warning message:
In utils::download.file(url = KEGGgraph::getKGMLurl(pathwayid = sub("hsa",  :
  downloaded length 47563 != reported length 0
```

**Comment by lshep** [comment]
have you updated to the latest versions of the packages?  We have fixed via this [PR][https://github.com/Bioconductor/KEGGREST/pull/9)  and should be fixed in 1.37.2 (devel Bioc 3.16)  and 1.36.2 (release Bioc 3.15).

**Comment by apelin20** [comment]
How embarrassing, did not expect myself to end up with an error due to outdated packages.

This worked for me:

```
library(devtools)
install_github("Bioconductor/KEGGREST")
```
